packages/block-library/src/navigation-link/edit.js

```diff
@@ -105,6 +117,39 @@ function NavigationLinkEdit( {
 		}
 	}, [ url ] );
 
+	useEffect( () => {
+		function handleDragStart( e ) {
+			// Check the first time when the dragging starts
+			handleDragEnter( e );
+		}
+
+		// Set isDragEnter to false whenever the user cancel the drag event by either releasing the mouse or press Escape.
+		function handleDragEnd() {
+			setIsDragEnter( false );
+		}
+
+		function handleDragEnter( e ) {
```

Comment:
Minor thing, but single-letter variables tend to be pretty rare in the codebase as the coding standards have a mention of avoiding abbreviations.

Reply:
Somehow I missed this!